### PR TITLE
RSDK-2012 & RSDK-2013 - Remove slam libraries

### PR DESCRIPTION
**Pull request by zaporter-work**
This is a stand-in PR to be completed at the end of the following tickets:
* https://viam.atlassian.net/browse/RSDK-2012
* https://viam.atlassian.net/browse/RSDK-2013

It deletes `slamLibraries` and associated AppImage creation as well as Readme information.

### CODE REVIEW

**Comment by zaporter-work** [comment]
LGTM

**Comment by kkufieta** [comment]
This PR can be merged after the brew packages are updated, and after this PR is merged: https://github.com/viamrobotics/slam/pull/167

The latter will allow the workflow checks to pass.

**Review by nicksanford — APPROVED** [review]

**Comment by nicksanford** [comment]
Looks like there are some tests / CI that also need to be resolved before merge.

**Comment by kkufieta** [comment]
> Looks like there are some tests / CI that also need to be resolved before merge.

@nicksanford Fixed!

File: README.md
Comment (kkufieta) [inline README.md:13]: @zaporter-work, we decided in one of the meetings not to link to our modules. I'll remove this again.

**Review by kkufieta — APPROVED** [review]
Let's do it!

**Review by tessavitabile — APPROVED** [review]
Yay! Two questions:
(1) I didn't see where this deletes app image creation. Is that in this PR? 
(2) Can someone please replace slam with the new repos in https://viam.atlassian.net/wiki/spaces/ENG/pages/94306305/Release+Process?

**Comment by kkufieta** [comment]
@tessavitabile 
> (1) I didn't see where this deletes app image creation. Is that in this PR?

That's already merged! https://github.com/viamrobotics/slam/pull/167

> (2) Can someone please replace slam with the new repos in https://viam.atlassian.net/wiki/spaces/ENG/pages/94306305/Release+Process?

Done!

**Comment by tessavitabile** [comment]
Thank you!

**Pull request merged**
